.editorconfig

```diff
@@ -0,0 +1,14 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
```

Comment:
We've had some issues on a project because of single quotes on SCSS files. Specifically while trying to target RTL where div[dir="rtl"] and div[dir=rtl] are valid syntax but div[dir='rtl'] isn't.

Might be worth overriding this for (S)CSS files.

Reply:
@msallent I've added `.scss` file to test this and the quote type "" will not be converted in Webstorm or VScode currently. Even when you execute the IDE reformatting options. So doesn't seem like a biggy, if the needs comes to it a dev would be able to add this override themselves.

Reply:
https://stylelint.io/user-guide/rules/string-quotes would overrule anything specified in the .editorconfig. Not sure if that's configured in your project?

But the .editorconfig should align with the quote-style we specify in our default linting config.